src/fileActions.jsx

```diff
@@ -394,6 +351,11 @@ export const EditPermissionsModal = ({ selected, path }) => {
     const groups = useFile("/etc/group", { syntax: etcGroupSyntax });
     const logindef = useFile("/etc/login.defs", { superuser: true });
 
+    if (!selected) {
+        const directory_name = path[path.length - 1];
+        selected = { name: directory_name, type: "dir" };
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-307-20240307-085857-be9ac358-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)